Doc/library/collections.rst

```diff
@@ -1256,12 +1256,18 @@ variants of :func:`functools.lru_cache`:
             if self.requests[args] <= self.cache_after:
                 self.requests.move_to_end(args)
                 if len(self.requests) > self.maxrequests:
-                    self.requests.popitem(0)
+                    self.requests.popitem(last=False)
             else:
+                # no longer need to keep track of how many times this
+                # entry has been seen; in a single-threaded environment,
```

Comment:
Please remove 1261 to 1265. The comments are getting in the way of reading the code.  Please stick to the substance of the original issue.

Reply:
Updated in commit aae218b39b27f09a39ed0cd6fba09cee26aafbea

Reply:
I'm reading through https://discuss.python.org/t/adopting-the-diataxis-framework-for-python-documentation/15072, and I'm wondering if we could add something along the lines of these comments elsewhere in the documentation (with a new corresponding issue).

Do you think it is worth it to include information about being careful about race conditions in multi-threaded environments, either at the top of this page (The Python Standard Library » Data Types» collections — Container datatypes) or in some other documentation page?

E.g. something like, "In a single-threaded environment, using the data structures provided by the `collections` module is straightforward, but in a multi-threaded environment, take care to think about how race conditions may cause unwanted behavior in an insertion, modification, or deletion operation for the data structure you are using."

Reply:
This is a really minor nuance.  Let's let it be.